README.md

```diff
@@ -1,258 +1,116 @@
-
-Those parts that were modified are located in ~/volttron_openadr_ven/volttron_openleadr.py. Future releases of OpenLeadr could potentially and adversely
-affect OpenADRVen agent if those releases directly or indirectly affect the modified code blocks. Thus, maintenance of this agent should closely monitor future changes to OpenLeadr.
-
-
-## Git Hooks and Pre-commit
-
-This repo provides pre-commit hooks to enforce code quality and standardization before pushing any commit to the repo. To use this tool locally,
-run the following command.
-
-```shell
-pre-commit install
-```
-
-To check that pre-commit scripts are installed and properly run, run pre-commit on all the files:
-
-```
-pre-commit run --all
+# Start platform with output going to volttron.log
+volttron -vv -l volttron.log &
 ```
 
-With pre-commit installed and configured, every time you run `git commit`, git will check your commit against a set of code linters. For details, see .pre-commit-config.yaml
-
-This project installs the following tools to aid in code standardization and best practices:
-
-## Black
-
-This repo uses [Black](https://pypi.org/project/black/), an opinionated Python code formatter. To use Black to check and automagically reformat your code,
-run the following command on your chosen Python file:
+Install and start the volttron-openadr-ven.
 
 ```shell
-black <path to file>
+vctl install volttron-openadr-ven --agent-config <path to agent config> --start
 ```
 
-## MyPy
+View the status of the installed agent
 
-This repo uses [MyPy](https://mypy.readthedocs.io/en/stable/), a static type checker for Python. To use MyPy on a Python file, run the following command:
 ```shell
-mypy <path to file>
+vctl status
 ```
 
-# CI/CD
-
-## Pull Requests
-
-Any pull requests to 'main' will be run through our 'test-pr.yml' Github Workflow, which will ensure that the requested code changes
-will be linted, tested, and built.
-
-## Release process
-
-NOTE: This section only applies to the maintainers of this repo.
-
-At a high level, the release process will consist of three steps:
-
-1. From 'main', update the project version and commit it to 'main'; also, create a tag from that commit.
-2. Manually update and publish the release notes.
-3. Publish to PyPi.
+# Agent Configuration
 
-Below is the step-by-step instructions on creating a release from your local environment:
+The required parameters for this agent are "ven_name" and "vtn_url". Below is an example of a correct configuration with optional parameters added.
 
-1. Fetch from the repo and ensure that your local 'main' is up-to-date with the remote 'main'
-
-```shell
-git fetch --all
-```
-
-2. Bump the project version in accordance with [Semantic Versioning](https://semver.org/). Use poetry to update the version in 'pyproject.toml'. For reference, see [poetry version docs](https://python-poetry.org/docs/cli/#version)
-
-```shell
-# command form
-# poetry version <version>
+```jsonpath
```

Comment:
what's the difference between ```json and ```jsonpath?  This is just json right?

Reply:
Actually I would rather use yaml in these so that comments are a first class citizen.